src/App.scss

```diff
@@ -37,12 +37,6 @@ $app-width: 900px;
   margin-right: 1rem;
 }
 
-.App-title {
-  font-size: 1.5em;
-  flex-grow: 1;
```

Comment:
I'm not sure how familiar you are with [CSS Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

To keep the logo on the far left, and the links on the far right, the item in the middle (in this case, the title) had a property of `flex-grow: 1` which means it fills the remaining space, thus pushing the other items apart.

Without this class, we no longer have that, but we should keep the links to the far right.

A couple of ways we can go forward with this:

1. Have the `App-links` container grow, and make sure the links are right aligned
2. Have the `App-image` container grow, and keep the image left aligned
3. Keep the `App-title` container, which pushes the logo and links apart, but keep it empty.

Personally, I prefer either 2 or 3

Reply:
I didn't appreciate Flexbox was being used - never heard of it (you know I'm not a UI kinda guy!).

Updating the `justify-content` property on the header container box seems to do the trick, running locally anyway.